Makefile

```diff
@@ -131,8 +131,19 @@ docker_push: docker_tag docker_auth ## push to docker hub
 	docker push 'openedx/blockstore:latest-newrelic'
 	docker push "openedx/blockstore:${GITHUB_SHA}-newrelic"
 
+COMMON_CONSTRAINTS_TXT=requirements/common_constraints.txt
+.PHONY: $(COMMON_CONSTRAINTS_TXT)
+$(COMMON_CONSTRAINTS_TXT):
+	wget -O "$(@)" https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt || touch "$(@)"
+
 upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
-upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
+upgrade: $(COMMON_CONSTRAINTS_TXT)	## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
+	sed 's/pyjwt\[crypto\]<2.0.0//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
```

Comment:
Should a first step and last step be added here to copy the *existing* `requirements/common_constraints.txt` file into a temp file and then restore it after all pip-compiles have run? Otherwise, a `make upgrade` will always modify the common constraints file according to a `git status`?

Reply:
The `requirements/common_constraints.txt` file will only get updated if there is some change in the actual constraint file here (https://github.com/edx/edx-lint/blob/master/edx_lint/files/common_constraints.txt)